native/thrift_consts.h

```diff
@@ -0,0 +1,65 @@
+#define STOP 0
```

Comment:
please add include guards, i.e.:

```
#ifndef THRIFT_CONSTS_H
... body of the include
#endif
```

see: https://stackoverflow.com/questions/1653958/why-are-ifndef-and-define-used-in-c-header-files